Index: src/main/modified/io/reactivex/super/io/reactivex/schedulers/Schedulers.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/modified/io/reactivex/super/io/reactivex/schedulers/Schedulers.java	(date 1536996481000)
+++ src/main/modified/io/reactivex/super/io/reactivex/schedulers/Schedulers.java	(date 1537088544000)
@@ -16,6 +16,7 @@
 import io.reactivex.Scheduler;
 import io.reactivex.annotations.GwtIncompatible;
 import io.reactivex.annotations.NonNull;
+import io.reactivex.gwt.schedulers.GwtSchedulers;
 import io.reactivex.internal.schedulers.*;
 import io.reactivex.plugins.RxJavaPlugins;

@@ -378,28 +379,28 @@
     static final class IOTask implements Callable<Scheduler> {
         @Override
         public Scheduler call() throws Exception {
-            throw new UnsupportedOperationException();
+            return GwtSchedulers.deferredScheduler();
         }
     }
 
     static final class NewThreadTask implements Callable<Scheduler> {
         @Override
         public Scheduler call() throws Exception {
-            throw new UnsupportedOperationException();
+            return GwtSchedulers.deferredScheduler();
         }
     }
 
     static final class SingleTask implements Callable<Scheduler> {
         @Override
         public Scheduler call() throws Exception {
-            throw new UnsupportedOperationException();
+            return GwtSchedulers.incrementalScheduler();
         }
     }
 
     static final class ComputationTask implements Callable<Scheduler> {
         @Override
         public Scheduler call() throws Exception {
-            throw new UnsupportedOperationException();
+            return GwtSchedulers.incrementalScheduler();
         }
     }
 }
